swap_meet/item.py

```diff
@@ -1,2 +1,21 @@
 class Item:
-    pass
+    def __init__(self, category="", condition=0):
+        self.category = category
+        self.condition = condition
+
+    def __str__(self):
+        return "Hello World!"
+
+    def condition_description(self):
```

Comment:
Good job having a case to handle conditions outside the expected range. But since condition is a float, what would happen if it was something like `3.5`? How would your current code handle this? Is there some other behavior we might prefer?